README.md

```diff
@@ -20,7 +20,25 @@ See the documentation for `zicht/cms` (https://github.com/zicht/cms)
 for more information on how the `zicht/page-bundle` relates to the
 other Zicht Bundles
 
+## Voters
+To enable the use of [voters](http://symfony.com/doc/current/security/voters.html) in your configuration you should properly 
+configure `security.yml` or in your config the `security` part
```

Comment:
- The part after "or" feels a bit wonky. Perhapse rewrite it to "or in the `security` part of your config." (Don't forget the dot at the end 😉 )